### DISCUSSION TITLE
TUYA zigbee TRV 0601

### DISCUSSION THREAD

**Discussion by janocha** [arendst/Tasmota#15104] [Q&A] [answered]
Hello,
I would like to ask if anyone knows the Mqtt topics related to the TUYA TRV 0601 head.
I connected it to my gate, I was able to change and read data such as set temperature, current temperature, but I would like to switch operating modes and I cannot find information about what commands to send.

**Comment by Jason2866** [comment, marked as answer]
There are infos https://github.com/Koenkk/zigbee2mqtt/issues/3821
